packages/studio-plugin/src/orchestrators/ParsingOrchestrator.ts

```diff
@@ -208,6 +215,15 @@ export default class ParsingOrchestrator {
     );
   };
 
+  private updateFilepathToDependencyTree(absPath: string) {
+    delete this.filepathToDependencyTree[absPath];
```

Comment:
do we need this `delete`? we reassign below anyways

Reply:
yes! we use `filepathToDependencyTree` to as "cached" info to speed up updating the dependency tree, so we delete it so we don't just update the tree with cached info.